templates/web/base/report/_inspect.html

```diff
@@ -10,6 +10,7 @@
         <p>
           <strong>[% loc('Report ID:') %]</strong>
           [% problem.id %]
+          <input type="hidden" name="report_id" value="[% problem.id | html %]">
```

Comment:
As this doesn't need submitting to the server (if I've read it right), could this instead be a span wrapping the printed out ID the line above, or something?